### PR TITLE
Issue1128 durack1 cython0.22 update

### PR DESCRIPTION
**Pull request by durack1**
@doutriaux1 @aashish24 there's no rush to merge this - 2.3 would be fine..

### CODE REVIEW

**Comment by durack1** [comment]
@doutriaux1 yep those damn travis_build and travis_submit could be the problem all along!

**Comment by durack1** [comment]
Useful info on travis CI build options: http://docs.travis-ci.com/user/build-timeouts/#Build-times-out-because-no-output-was-received

**Comment by durack1** [comment]
@doutriaux1 seems like timeouts are more explicitly the problem now:
```
Still running (20 of 20): ctest -VV -S ../uvcdat/CMake/travis_build.cmake
Timeout (20 minutes) reached. Terminating "ctest -VV -S ../uvcdat/CMake/travis_build.cmake"
The command "travis_wait ctest -VV -S ../uvcdat/CMake/travis_build.cmake" exited with 1.
```

**Comment by durack1** [comment]
@aashish24 I think your expertise is required here - seems like the travis config needs some tweaks..

File: .travis.yml [hunk 1]
```diff
@@ -16,6 +16,5 @@ script:
     - cd ..
     - mkdir _build
     - cd _build
-    - cmake -DGIT_PROTOCOL=git:// -DCDAT_BUILD_MODE=LEAN -DCDAT_BUILD_GRAPHICS=ON -DCDAT_BUILD_SCIPY=OFF ../uvcdat
-    - ctest -VV -S ../uvcdat/CMake/travis_build.cmake
-    - ctest -VV -S ../uvcdat/CMake/travis_submit.cmake
```
Comment (aashish24) [inline .travis.yml]: Please don't change travis options. I explicitly set them for a reason. The travis has a timeout so we can only test lean mode for now. Also, I would recommend having a discussion over the email first -:)

File: .travis.yml [hunk 2]
```diff
@@ -16,6 +16,5 @@ script:
...
+    - travis_wait cmake -DGIT_PROTOCOL=git:// -DCDAT_BUILD_MODE=DEFAULT -DCDAT_BUILD_GUI=OFF -DCDAT_BUILD_SCIPY=OFF ../uvcdat
+    - travis_wait ctest -VV -S ../uvcdat/CMake/travis_build.cmake
```
Comment (aashish24) [inline .travis.yml]: Let's not run test on travis. I am working on a different solution until then let's not change the travis

**Comment by aashish24** [comment]
@durack1 could you please undo travis changes for now. Currently the travis is set only for lean mode as travis has time limitations. I am working on another solution but in the mean time, if you want to test then you can setup a dashboard that can test a particular branch say "integration". Let me know if you need more help on that.

**Comment by durack1** [comment]
@aashish24 I don't think this is true, with travis ci we have 120 mins.. It also seems that the problem is more related to timeouts within the install (no activity for 20 mins) rather than exceeding the 120 min limit, when reviewing the logs of the failed builds this seems to be the problem

**Comment by durack1** [comment]
@aashish24 following the log errors this is what we've tried: http://docs.travis-ci.com/user/build-timeouts/#Build-times-out-because-no-output-was-received

**Comment by durack1** [comment]
@aashish24 so this is the latest build, with both travis_build.cmake and travis_submit.cmake back in (using `LEAN` build mode):
```
The command "travis_wait cmake -DGIT_PROTOCOL=git:// -DCDAT_BUILD_MODE=LEAN -DCDAT_BUILD_GUI=OFF -DCDAT_BUILD_SCIPY=OFF ../uvcdat" exited with 0.
$ travis_wait ctest -VV -S ../uvcdat/CMake/travis_build.cmake
Still running (20 of 20): ctest -VV -S ../uvcdat/CMake/travis_build.cmake
Timeout (20 minutes) reached. Terminating "ctest -VV -S ../uvcdat/CMake/travis_build.cmake"
The command "travis_wait ctest -VV -S ../uvcdat/CMake/travis_build.cmake" exited with 1.
$ travis_wait ctest -VV -S ../uvcdat/CMake/travis_submit.cmake
Still running (1 of 20): ctest -VV -S ../uvcdat/CMake/travis_submit.cmake
The command "travis_wait ctest -VV -S ../uvcdat/CMake/travis_submit.cmake" exited with 0.
```

**Pull request closed without merging**